src/sonic-yang-models/yang-models/sonic-gnmi.yang

```diff
@@ -85,6 +85,11 @@ module sonic-gnmi {
                     type uint32;
                     description "Certificate revocation list cache expire duration.";
                 }
+
+                leaf user_auth {
+                    type string;
```

Comment:
Could you use pattern? user_auth can be jwt, password and cert

Reply:
Fixed